AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift

```diff
@@ -34,3 +34,4 @@ struct ApiConfig {
         return parameters
     }
 }
+
```

Comment:
why, oh, why ..

this is careless. this implies there is no review of the diff prior to submitting. bad shresth, no cookie.